File: PrimeiraEtapa/Semana7/CollectionExample/CollectionExample/ViewController.m
//
//  ViewController.m
//  CollectionExample
//
//  Created by bepid on 24/11/14.
//  Copyright (c) 2014 BEPiD. All rights reserved.
//

#import "ViewController.h"
#import "Cell.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
	// Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recr    eated.
}

-(NSInteger)numberOfSectionsInCollectionView:(UICollectionView *)collectionView{
    return 5;

}

-(NSInteger)collectionView:(UICollectionView *)collectionView numberOfItemsInSection:(NSInteger)section{
    return 6;
}

-(UICollectionViewCell *)collectionView:(UICollectionView *)collectionView cellForItemAtIndexPath:(NSIndexPath *)indexPath{
    Cell *celula = [collectionView dequeueReusableCellWithReuseIdentifier:@"celula" forIndexPath: indexPath];
    celula.mLabel.text = [NSString stringWithFormat:@"{%ld, %ld}", (long) indexPath.section, (long) indexPath.row ];
    return celula;
}



@end

File: Quinta Etapa/NSLocalizedDemo/NSLocalized-Demo/ViewController.swift
//
//  ViewController.swift
//  NSLocalized-Demo
//
//  Created by Diego Vidal on 18/08/15.
//  Copyright (c) 2015 Diego Vidal. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var labelCountry: UILabel!
    @IBOutlet weak var imageOutlet: UIImageView!
    @IBOutlet weak var currencyOutlet: UILabel!
    @IBOutlet weak var valueOutlet: UILabel!
    @IBOutlet weak var myMessageOutlet: UITextView!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        labelCountry.text = NSLocalizedString("countryKey", comment: "country")
        imageOutlet.image = UIImage(named: NSLocalizedString("flagKey", comment: "flag"))
        valueOutlet.text = NSLocalizedString("valueKey", comment: "value")
        myMessageOutlet.text = NSLocalizedString("helloKey", comment: "message")
        
        let local = NSLocale(localeIdentifier: NSLocalizedString("localKey", comment: "local"))
        currencyOutlet.text = local.objectForKey(NSLocaleCurrencySymbol) as? String
        
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

